package.json

```diff
@@ -21,14 +22,17 @@
     "*.{js,jsx,ts,tsx}": [
       "cross-env NODE_ENV=development eslint --cache"
     ],
-    "*.json,.{eslintrc,prettierrc}": [
-      "prettier --ignore-path .eslintignore --parser json --write"
+    "*.json": [
+      "prettier --write"
+    ],
+    ".{eslintrc,prettierrc}": [
+      "prettier --write"
```

Comment:
I separated this into two rules because I tested and `"*.json,.{eslintrc,prettierrc}"` appeared to be invalid. (It didn't match any files.)